docs/process/style_guide.md

```diff
@@ -608,13 +608,12 @@ Main issue: [#113](https://github.com/unicode-org/icu4x/issues/113)
 
 When structs with public string fields contain strings, use the following type conventions:
 
-- If lifetime parameters are allowed, use `&'a str`.
-- If lifetime parameters are not allowed, use one of:
-  - [TinyStr](https://github.com/zbraniecki/tinystr) if the string is ASCII-only.
-  - [SmallString](https://crates.io/crates/smallstr) for shorter strings, with a stack size ∈ {8, 12, 16, 20} that fits at least 99% of cases.
-  - `Cow<'static, str>`  for longer strings.
+- `&'s str` if the struct does not need to own the string.
+- `Cow<'s, str>`  if the string could be borrowed or owned.
+- [TinyStr](https://github.com/zbraniecki/tinystr) if the string is ASCII-only.
+- [SmallString](https://crates.io/crates/smallstr) for shorter strings, with a stack size ∈ {8, 12, 16, 20} that fits at least 99% of cases.
 
-A case where lifetime parameters are not allowed is in the data provider structs.  TinyStr, SmallString, or `Cow<'static, str>` should be used as appropriate in data provider structs.
+In general, do not use `String` in structs; use `Cow<'s, str>` instead.
```

Comment:
This statement could use an explanation as to "why". I believe the "why" here is that it's preferable to be able to borrow strings, as this avoids unnecessary duplication of strings.